src/libs/ReportUtils.ts

```diff
@@ -673,6 +674,13 @@ function isChatReport(report: OnyxEntry<Report> | EmptyObject): boolean {
     return report?.type === CONST.REPORT.TYPE.CHAT;
 }
 
+/**
+ * Checks if a report is an invoice report.
```

Comment:
This comment isn't needed because the function name is self explanatory

Reply:
Done - c852145.